core/controllers/android_e2e_config_test.py

```diff
@@ -76,20 +75,23 @@ def test_check_if_topic_is_published(self):
         story.validate()
         topic.validate(strict=True)
         exploration.validate(strict=True)
-        for node in story.story_contents.nodes:
-            self.assertTrue(node.exploration_id == EXPLORATION_ID)
```

Comment:
It seems useful to still verify that the story has the correct chapter in it--I suggest adding an assertion for that case.

Reply:
Ah. I forgot to add this, Thanks 

Done.